flasksrc/schema.sql

```diff
@@ -9,6 +9,8 @@ DROP TABLE IF EXISTS toynet_surveys;
 DROP TABLE IF EXISTS toynet_survey_questions;
 DROP TABLE IF EXISTS toynet_survey_options;
 DROP TABLE IF EXISTS toynet_survey_types;
+DROP TABLE IF EXISTS toynet_topos;
+DROP TABLE IF EXISTS toynet_sessions;
```

Comment:
I'd like this at the top under user table(s) (will be multiple when I merge my PR) and with a newline between these lines and other lines.